libs/langchain/langchain/__init__.py

```diff
@@ -370,7 +370,6 @@ def __getattr__(name: str) -> Any:
 
 __all__ = [
     "LLMChain",
-    "LLMBashChain",
```

Comment:
Forgot to do it last time -- this fixes auto-completion in the ipython repl

Reply:
should we remove LLMMathChain below too

Reply:
It's still in langchain, what we did was make the underlying dependency optional (and up to the user) and bumped it to a minimum version that contains sanitization